aptos-move/e2e-move-tests/src/tests/mint_nft.rs

```diff
@@ -204,3 +204,47 @@ fn sample_mint_nft_signature() {
         mint_proof_signature2
     );
 }
+
+/// run `cargo test sample_tutorial_signature -- --nocapture`
+/// to sample a valid signature for calling `resource_account_address::minting::mint_nft()`
+/// in `aptos-move/move-exampels/mint_nft`
+#[test]
+fn sample_tutorial_signature() {
+    let mut h = MoveHarness::new();
+
+    // supply the resource_address
+    let resource_address = h.new_account_at(AccountAddress::from_hex_literal("0xcafe").unwrap());
```

Comment:
where is the place for creating a resource account and deploy the contract?

Reply:
see an example in the comments [here](https://github.com/aptos-labs/aptos-core/blob/c2020c3227d1ceb7cf1a98347d663d1e0edff2b9/aptos-move/move-examples/mint_nft/sources/minting.move#L27)!

Reply:
this function is just for generating a valid signature to supply to the CLI